duva/src/domains/caches/actor.rs

```diff
@@ -91,6 +91,28 @@ impl CacheActor {
         Ok(())
     }
 
+    pub(crate) fn append(
```

Comment:
try this:

```rust
let val = self.cache.entry(key.clone().or_insert_with(||"".to_string());
val.value.push_str(value.as_str());

```

Reply:
@Migorithm

I've simplified append implementation with mentioned suggestion. :)